README.md

```diff
@@ -78,7 +78,7 @@ Eg: `curl ... | NVM_DIR="path/to/nvm"`. Ensure that the `NVM_DIR` does not conta
 command -v nvm
 ```
 
-simply close your current terminal, open a new terminal, and try verifying again.
+simply close your current terminal, open a new terminal, and try verifying again, or type and run the command `. ~/.bashrc`.
```

Comment:
The trick here is that the file might not be ".bashrc", depending on the shell someone is using.

Reply:
Ah yes, true.  Rejecting and closing, doesn't making sense to even go down this path.